src/global.zig

```diff
@@ -26,19 +27,14 @@ pub const GlobalState = struct {
     gpa: ?GPA,
     alloc: std.mem.Allocator,
     action: ?cli.Action,
-    logging: Logging,
     rlimits: ResourceLimits = .{},
+    log_level: std.log.Level,
+    log_enabled: bool,
```

Comment:
I don't know if we want to use a boolean here. I think errors should be logged even in quiet mode, so that users may know when something truly *bad* happens. We don't use them nearly as much as warnings in the codebase, and the places where they're found are usually situations that have gone quite wrong and should never happen.

Reply:
Fair enough.  I can remove the `-q/--quiet` option.

Reply:
`-q/--quiet` has been removed.